### ISSUE TITLE
Add prettier config

### ISSUE THREAD

**Issue by corasaurus-hex** [clavascript/clavascript#18]
It would be nice to have automatic and consistent formatting for js code as we write it.

https://prettier.io/

**Comment by borkdude** [comment]
I'm fine with this. I just don't get some decisions but if I can setup my emacs that it will do this for me, I will cave.

**Comment by borkdude** [comment]
Already installed, geez, that was easy:

```
npm install -g prettier
package-install prettier-js
M-x prettier-js-mode
```

**Comment by borkdude** [comment]
@corasaurus-hex I'm not sure what needs to be done before closing this issue

**Comment by corasaurus-hex** [comment]
We can add a prettierrc file into the root of the project which prettier will pick up and use so that we get consistent formatting across devs and across editors.

**Comment by corasaurus-hex** [comment]
We can also add prettier as a dev dependency so that everyone gets a consistent version for formatting.
